PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,214 @@
+
+            So for example "Get-Process" will retrieve all the processes running on 
+            your current machine.
+        #>
+
+        It 'is for commands that retrieve data'  {
+            <#
+                Using the Get-Command cmdlet, which retrieves all available commands,
+                find 5 new Get-* commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Get cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Get).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.count -eq 5 | 
+            Should -BeTrue -Because "five unique cmdlets should be supplied"
+        }
+    }
+
+    Context "New" {
+         <#
+            New
+
+            Cmdlets with the New verb are used to create data.
+
+            So for example 'New-GUID' will create a new GUID, or 'New-LocalUser' will create a local
+            user on your machine.
+
+            Before continuing with this exercise, try using the following code to create a file:
+
+                $Path = "YOUR PATH.txt"
+                New-Item -Path $ItemPath -ItemType file
+
+            This will create a new text file, in the location you specify. 
+        #>
+
+        It 'is for commands that create data' {
+            <#
+                Using the Get-Command cmdlet, which all available commands,
+                find 5 New-* new commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a New cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb New).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.count -eq 5 | 
+            Should -BeTrue -Because "five unique cmdlets should be supplied"
+
+        }
+
+    }
+
+    Context "Add" {
+        <#
+            Add
+
+            Cmdlets with the Add verb append data to an existing object or data source.      
+
+            If you followed the example in the 'new' test, you can use Add-Content to add some
+            text to your newly created text file:
+            
+                $Path = "YOUR PATH.txt"
+                Add-Content -Path $Path -Value "Sgt. Bash is the best house robot because... fire."
+            
+            Before continuing, run this command several times. See what happens; is it the
+            result you expected?
+
+            You'll see that several lines of text were added to the file. This cmdlet only appends
+            information, it does not overwrite. 
+
+            Basically, if it doesn't exist then a cmdlet with the add verb can probably be used
+            to make it so.
+
+            A common example an Operations Engineers may see is with Office 365 calendar permissions. 
+            If an Engineer wants to grant permissions to somebody who doesn't have any, they use Add.
```

Comment:
```suggestion
            If you want to grant permissions to somebody who doesn't have any, you use a cmdlet with the Add verb.
```

Reply:
I suppose you could also just reference the specific cmdlet you're talking about here rather than being overly general in what is otherwise a reasonably targeted example.

Reply:
🦄